mavisp/modules.py

```diff
@@ -595,11 +595,19 @@ def ingest(self, mutations):
             this_error = f"The variants_output.csv file must contain the interpretation column"
             raise MAVISpMultipleError(warning=warnings,
                                       critical=[MAVISpCriticalError(this_error)])
-
-        clinvar_found[id_col] = clinvar_found[id_col].astype(str)
-        clinvar_found = clinvar_found.groupby('variant_name').agg(lambda x: ", ".join(list(x)))[[id_col, 'interpretation']]
-        self.data = clinvar_found.rename({ id_col          : 'Clinvar Variation ID',
-                                          'interpretation' : 'ClinVar interpretation'}, axis=1)
+            
+        try:            
+            clinvar_found[id_col] = clinvar_found[id_col].astype(str)
```

Comment:
This line doesn't need to be in this code block - it is executed in the same way regardless, so we can have it only once before the `if`/`try` and avoid code duplication